notebooks/notebooks/paper/defense-compute.ipynb

```diff
@@ -12,7 +12,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #80.    gpu_type_to_a6000_gpu_days = {'NVIDIA H100 80GB HBM3': 0.36900884742276013, 'NVIDIA RTX A6000': 1.0}
</pre><p>I don't think these changes are actually necessary for this PR but I was first going sequentially through the notebook and trying to make it usable on VAST.</p><p> </p><p>See code comment above this for an explanation of why things changed here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlignmentResearch/KataGoVisualizer/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>